### PR TITLE
Add richtext to block configuration

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [EZEE-2691](https://jira.ez.no/browse/EZEE-2691)
| Versions      | master, 2.5

**added:**
- `richtext` attribute type to Extending the Page section
- Creating block with `richtext` attribute type section to Cookbook

### CODE REVIEW

**Comment by michalmilc90** [comment]
Zrzut ekranu 2019-05-08 o 12 32 45

File: docs/cookbook/richtext_block.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        
+    ```
+    Here, the line 2 is responsible for rendering the content from XML to HTML5.
+        
+    Proceed with creating a separate `app/Resources/views/blocks/my_block/config.html.twig` template:
+    
+    ``` html+twig
+    {% extends '@EzPlatformPageBuilder/page_builder/block/config.html.twig' %}
+             
+            {% block meta %}
```
[the remark this answers is not in the document]
Comment (michalmilc90) [reply docs/cookbook/richtext_block.md]: I will replicate the indentation from this this [config file](https://github.com/ezsystems/ezplatform-page-builder/blob/master/src/bundle/Resources/views/page_builder/block/config/richtext.html.twig)

File: mkdocs.yml [hunk 2]
```diff
@@ -143,7 +143,7 @@ pages:
         - 'API search result pagination': 'cookbook/paginating_api_search_results.md'
         - 'AWS S3 clustering': 'cookbook/setting_up_amazon_aws_s3_clustering.md'
         - 'Binary attachments via REST API': 'cookbook/creating_content_with_binary_attachments_via_rest_api.md'
-        - 'Creating block with `richtext` attribute type': 'cookbook/richtext_block.md'
+        - 'Creating block with richtext attribute type': 'cookbook/richtext_block.md'
```
Comment (MagdalenaZuba) [inline mkdocs.yml]: The title is still too long for a TOC

Comment (michalmilc90) [reply mkdocs.yml]: I know, I haven't pushed the changes yet :D

File: docs/guide/extending_page.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Extending the Page
+# Creating custom Page blocks
 
 !!! enterprise
```
Comment (DominikaK) [inline docs/guide/extending_page.md]: You can now get rid of the second heading (can't leave a comment at the proper line)

File: docs/guide/extending_page.md [hunk 4]
```diff
@@ -80,6 +82,7 @@
     |`string`|String|-|
     |`url`|URL|-|
     |`text`|Text block|-|
+    |`richtext`|Rich text block (see [creating richtext block](/guide/richtext_block.md)|-|
```
Comment (DominikaK) [inline docs/guide/extending_page.md]:
```suggestion
    |`richtext`|Rich text block (see [creating richtext block](richtext_block.md)|-|
```

File: docs/guide/richtext_block.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
+# Creating custom richtext block
+
+!!! enterprise
+
+    To create a richtext [custom Page block](/guide/extending_page.md), you need to define its layout, provide templates, add a subscriber and register it as a service.
```
Comment (DominikaK) [inline docs/guide/richtext_block.md]:
```suggestion
    To create a richtext [custom Page block](extending_page.md), you need to define its layout, provide templates, add a subscriber and register it as a service.
```

File: docs/guide/richtext_block.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+    Complete the procedure with registering the subscriber as a service:
+            
+    ``` yaml
+    services:
+        AppBundle\Event\Subscriber\RichTextBlockSubscriber:
+            tags:
+                - { name: kernel.event_subscriber }
+    ```
+        
+    For details on customizing additional options of the block or creating custom blocks with other attribute types, see [extending the Page](/guide/extending_page.md).
```
Comment (DominikaK) [inline docs/guide/richtext_block.md]:
```suggestion
    For details on customizing additional options of the block or creating custom blocks with other attribute types, see [Creating custom Page blocks](/guide/extending_page.md).
```

File: docs/guide/richtext_block.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
+# Creating custom richtext block
```
Comment (DominikaK) [inline docs/guide/richtext_block.md]:
```suggestion
# Creating custom richtext blocks
```
(for consistency)

**Pull request merged**